### PR TITLE
Initial overview for class messaget.

### PR DESCRIPTION
**Pull request by sonodtt**
- [ ] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- [x] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

Initial overview for class messaget.

### CODE REVIEW

**Review by chrisr-diffblue — APPROVED** [review]

**Review by xbauch — APPROVED** [review]

File: src/util/message.h [hunk 1]
```diff
@@ -118,6 +118,18 @@ class stream_message_handlert:public message_handlert
   std::ostream &out;
 };
 
+/// \brief Class that provides messages with a built-in verbosity 'level'.
```
Comment (peterschrammel) [inline src/util/message.h:121]: `messages` -> `message output` or `message logging`

File: src/util/message.h [hunk 2]
```diff
@@ -118,6 +118,18 @@ class stream_message_handlert:public message_handlert
...
+//// A message handler (of type \ref message_handlert) outputs messages only
+/// to a specific level of verbosity. By default the verbosity filtering level
+/// is set to the maximum level (10) - all messages printed (level 10 messages are
+/// debug information).
+/// Common practice is to inherit from the \ref messaget class, to provide
+/// local infrastructure for messaging, by calling one of the utility
+/// methods, e.g. debug(), warning() etc. - which return a reference to a new
```
Comment (peterschrammel) [inline src/util/message.h]: Put code elements in \` \`

**Review by peterschrammel — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: 971960748e7bfdf0f29d7cea552728414b76bc6f).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/91359884

**Comment by tautschnig** [comment]
I believe this is good to go - @sonodtt, would you mind squashing your commits and remove the "[ci skip]" suffix to the commit message?

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: b726fa6a1488a595e87557681d982d3f8cea8e86).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/91537290

**Pull request merged**